platform/platform.go

```diff
@@ -0,0 +1,18 @@
+package platform
```

Comment:
I believe we can go without adding this package

Reply:
@viveksyngh please can you answer this?

Reply:
`platform` package has been remove and we are using `mainPlatform` variable for default value.